Makefile

```diff
@@ -13,6 +13,11 @@ CFLAGS += -D ENABLE_SDL
 CFLAGS += `sdl2-config --cflags`
 LDFLAGS += `sdl2-config --libs`
 
+# Whether to enable computed goto in riscv.c
+ifeq ("$(NO_COMPUTED_GOTO)", "")
+riscv.o: CFLAGS += -fno-gcse -fno-crossjumping -D ENABLE_COMPUTED_GOTO
```

Comment:
The option `-fno-crossjumping` is gcc specific. And, clang would reject it.
See https://github.com/peberlein/interpreter